### PR TITLE
Conform verbiage example to our expectations

### PR DESCRIPTION
**Pull request by billygriffin**
Update docs to make the example present tense. cc: @nerdneha

### CODE REVIEW

File: docs/process/writing-release-notes.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ We don't include release notes that don't impact the user's experience using Des
 We'll make exceptions for security vulnerability fixes, especially if they're high profile:
```
Comment (nerdneha) [inline docs/process/writing-release-notes.md:29]: How do we feel about being explicit and saying "While we won't mention the CVE, we will mention where the security vulnerability was" -- too explicit? It's come up twice so I wondered if it'd be helpful to memorialize

Comment (billygriffin) [inline docs/process/writing-release-notes.md:29]:
```suggestion
We'll make exceptions for security vulnerability fixes, especially if they're high profile. These are typically general in nature and do not include the CVE for the vulnerability:
```

Comment (billygriffin) [reply docs/process/writing-release-notes.md:29]: How's that @nerdneha?

Comment (nerdneha) [reply docs/process/writing-release-notes.md:29]: Fantastic! 🚢

Comment (billygriffin) [reply docs/process/writing-release-notes.md]: Done!

**Pull request merged**